src/ngraph/op/util/broadcasting.cpp

```diff
@@ -153,16 +166,35 @@ static std::shared_ptr<ngraph::Node>
     }
 
     // Remove axes which have length of 1 from source shape
-    auto broadcasted_node = std::make_shared<ngraph::op::Reshape>(
-        node, ngraph::get_default_order(node->get_shape()), squeezed_shape);
+    ngraph::Output<ngraph::Node> broadcasted_value = std::make_shared<ngraph::op::Reshape>(
```

Comment:
Out of scope for this PR, but why are we not `using namespace ngraph` in this file?

Reply:
I wondered the same. It's not alone.